packages/block-editor/src/components/line-height-control/index.native.js

```diff
@@ -20,7 +21,7 @@ export default function LineHeightControl( { value: lineHeight, onChange } ) {
 			step={ STEP }
 			value={ value }
 			onChange={ onChange }
-			units={ false }
+			units={ [] }
```

Comment:
Based on the [separate index](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/index.native.js#L56), I'd assume this is the RN version of `UnitControl`, so we probably _shouldn't_ make this change? Like, I see some [runtime changes](https://github.com/WordPress/gutenberg/blob/08625a7be45d72c37bd4cc39a00a9ac68577bf73/packages/components/src/unit-control/index.native.js#L119-L148) that would happen.